software/donetick.yml

```diff
@@ -0,0 +1,13 @@
+name: "Donetick"
+website_url: "https://donetick.com"
+source_code_url: "https://github.com/donetick/donetick"
+description: "Task and chore management tool for personal and family use. with advanced scheduling, flexible assignment, and group sharing capabilities, detailed history, automation via API. with simple and modern design."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Go
+  - Docker
+tags:
+  - Task Management & To-do Lists
+depends_3rdparty: false
```

Comment:
```suggestion
```
This can be excluded completely, as it by default false

Reply:
sounds good! omitted!